app/models/reviews-model.js

```diff
@@ -42,15 +42,85 @@ exports.updateReviewVotes = (inc_votes, review_id) => {
     });
 };
 
-exports.selectReviews = () => {
-  return connection
-    .query(
-      `
+exports.selectReviews = (query = { sort_by: "created_at" }) => {
+  let methods = ["sort_by", "order", "category"];
+  let searchSort_by = "created_at";
+  let searchOrder = "DESC";
+  let whereStr = "";
+  const request = Object.keys(query);
+  if (!methods.includes(request[0])) {
```

Comment:
why are you only checking the first query here?

what if it was:
```
GET /api/reviews?sort_by=votes&danger=zone
```
?

Reply:
Edit to change to a for loop.  Had some issues with my promise.reject not preventing the rest of the code from working, so wrapped it all in a big if else.